src/components/templates/Camp/hooks/useMapGrid.tsx

```diff
@@ -9,8 +9,10 @@ import { MapRow } from "../../../molecules/MapRow";
 interface UseMapGrid {
   showGrid?: boolean;
   userUid?: string;
-  columnsArray: JSX.Element[];
-  rowsArray: JSX.Element[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  columnsArray: any[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  rowsArray: any[];
```

Comment:
Definitely don't want to be using `any`/disabling the eslint rules like this (assuming this is more than just a temporary 'work in progress' change)